bio2zarr/vcf2zarr/vcz.py

```diff
@@ -313,7 +321,7 @@ def fixed_field_spec(
             array_specs.append(spec_from_field(field))
 
         if gt_field is not None:
-            ploidy = gt_field.summary.max_number - 1
+            ploidy = max(gt_field.summary.max_number - 1, 1)
```

Comment:
This is needed since we can't have a chunk size of zero (below), but also I don't think a ploidy of zero makes sense?

Reply:
Yep, makes sense.